### PR TITLE
fixed overflow for review page and added submit button

### PR DESCRIPTION
**Pull request by hannakohxt**

### CODE REVIEW

File: client/src/styles/components/_profile.scss [hunk 1]
```diff
@@ -49,4 +49,10 @@
 		overflow-x: auto;
 	}
 
+	button {
```
Comment (rong2) [inline client/src/styles/components/_profile.scss]: There's already a button style defined in the component directory. Do you need to define a new style? If you do, please define the style there.

Comment (hannakohxt) [inline client/src/styles/components/_profile.scss]: Oh it's because i made it smaller. Should I not?

Comment (rong2) [inline client/src/styles/components/_profile.scss]: You can, but I would still define it as a class and put your styling for your button in the _buttons.scss

**Comment by hannakohxt** [comment]
Is the styling for the button okay now?

**Pull request merged**